tic_tac_toe/game.py

```diff
@@ -24,7 +26,9 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
+    if isinstance(position, tuple) and len(position) == 2 and position[0] in [0, 1, 2] and position[1] in [0, 1, 2]:
```

Comment:
This line is too long. Pep8 recommends less than 80 characters per line.
Please make it shorter by indenting the `if ` statement properly.